config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -234,6 +234,8 @@
          files="src[\\/]xdocs[\\/]checks[\\/]coding[\\/]nofinalizer.xml.template"/>
   <suppress id="propertiesMacroMustExist"
          files="src[\\/]xdocs[\\/]checks[\\/]coding[\\/]overloadmethodsdeclarationorder.xml.template"/>
+  <suppress id="propertiesMacroMustExist"
+            files="src[\\/]xdocs[\\/]checks[\\/]coding[\\/]constructorsdeclarationgrouping.xml.template"/>
```

Comment:
Align indentation of attribute with other lines like this

Reply:
done